src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -138,12 +138,6 @@ struct
       Frozen_ledger_hash.of_ledger_hash
       @@ Ledger.merkle_root (Ledger.of_database root_snarked_ledger)
     in
-    assert (
```

Comment:
I approved, but why is this being removed? This seems like a worthwhile check to have.